Create rule S6595: Package update should not be executed without installing it (#1757)
SonarSource/rspec: rules/S6595/docker/rule.adoc

Running `update` of your package manager in a single `RUN` instruction stores the cache index in the file system.

> This cache is not needed for the good working of the installed software.

Reviewer: ```suggestion
This cache is not needed for the installed software to work properly.
```
